data/json/professions.json

```diff
@@ -971,6 +971,341 @@
+          { "item": "money_five" },
+          { "item": "money_five" },
+          { "item": "money_ten" },
+          { "item": "money_ten" },
+          { "item": "money_ten" },
+          { "item": "money_twenty" },
+          { "item": "money_twenty" },
+          { "item": "money_twenty" },
+          { "item": "coin_half_dollar" },
+          { "item": "coin_half_dollar" },
+          { "item": "coin_half_dollar" },
+          { "item": "coin_half_dollar" },
+          { "item": "coin_half_dollar" },
+          { "item": "coin_penny" },
+          { "item": "coin_penny" },
+          { "item": "coin_penny" },
+          { "item": "coin_penny" },
+          { "item": "coin_penny" },
+          { "item": "coin_dime" },
+          { "item": "coin_dime" },
+          { "item": "coin_dime" },
+          { "item": "coin_dime" },
+          { "item": "coin_dime" },
+          { "group": "charged_smart_phone" }
+        ]
+      },
+      "male": {
+        "entries": [
+          { "item": "boxer_shorts" }
+        ]
+      },
+      "female": {
+        "entries": [
+          { "item": "hairpin" },
+          { "item": "bra" },
+          { "item": "panties" }
+        ]
+      }
+    },
+    "age_lower": 16
+  },
+  {
+    "type": "profession",
+    "id": "manager",
+    "name": "Store Manager",
+    "description": "You used to be the one holding the keys, balancing inventory, scheduling shifts, setting up and down the store's alarm system, and dealing with the occasional disgruntled employee or the unhappy customer.  When everything fell apart, you lost everything: Family, employees, your store.  Can you navigate the Cataclysm with the same determination that once kept your store running smoothly?",
+    "points": 1,
+    "skills": [
+      { "level": 2, "name": "speech" },
+      { "level": 1, "name": "computer" },
+      { "level": 1, "name": "devices" }
+    ],
+    "items": {
+      "both": {
+        "entries": [
+          { "item": "polo_shirt", "variant": "s_polo" },
+          { "item": "wristwatch" },
+          { "item": "fanny" },
+          { "item": "pants", "variant": "pants_black" },
+          { "item": "socks" },
+          { "item": "dress_shoes" },
+          { "item": "old_key" },
+          { "item": "gum", "variant": "gum_peppermint" }
+          { "item": "gum", "variant": "gum_peppermint" }
+          { "item": "gum", "variant": "gum_peppermint" }
+          { "item": "gum", "variant": "gum_peppermint" }
+          { "item": "gum", "variant": "gum_peppermint" }
+          { "item": "pen", "ammo-item": "blue_pen_ink", "charges": 100 },
+          { "group": "charged_smart_phone" }
+        ]
+      },
+      "male": {
+        "entries": [
+          { "item": "boxer_briefs" }
+        ]
+      },
+      "female": {
+        "entries": [
+          { "item": "hairpin" },
+          { "item": "bra" },
+          { "item": "panties" }
+        ]
+      }
+    },
+    "age_lower": 21
+  },
```

Comment:
These all have the same problem, professions are "roleplay hooks", but they need to be mechanically distinct in order to support that, we don't need dozens of different practically identical professions only separated by their descriptions.
```suggestion
```